### PR TITLE
add gateway on CNS to get populated

### PR DESCRIPTION
**Pull request by paulyufan2**
**Reason for Change**:

This PR is to add ip and gateway on CNS to get populated

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

File: network/network_windows.go [hunk 1]
```diff
@@ -304,6 +304,7 @@ func (nm *networkManager) configureHcnNetwork(nwInfo *EndpointInfo, extIf *exter
 
 	// Populate subnets.
 	for _, subnet := range nwInfo.Subnets {
+		logger.Info("NextHop is", zap.String("NextHop", subnet.Gateway.String()))
```
Comment (tamilmani1989) [inline network/network_windows.go]: remove log.. we log when we receive from cns right?

**Review by tamilmani1989 — APPROVED** [review]

File: network/network_windows.go (hunk 1 above)
Comment (paulyufan2) [reply network/network_windows.go]: logs are removed

File: .pipelines/pipeline.yaml [hunk 2]
```diff
@@ -208,6 +208,11 @@ stages:
                 name: azure-ipam
                 os: windows
                 os_version: ltsc2022
+              cni_dropgz_windows2025_amd64:
+                arch: amd64
+                name: cni-dropgz
+                os: windows
+                os_version: ltsc2022-KB5035857
```
[the remark this answers is not in the document]
Comment (paulyufan2) [reply .pipelines/pipeline.yaml]: yes removed them from this PR and we will push windows 2025 image if we have official one

**Pull request merged**